uilib/src/components/Loader/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Svg, Circle, G, Rect } from "react-native-svg";
+import { useTheme } from "styled-components/native";
+import { TouchableOpacity } from "react-native";
+
+type Props = {
+  // float number between 0 and 1
+  progress?: number;
+
+  // function triggered when pressing the loader
+  onPress: () => void;
+
+  // Display the square in the middle of the loader
+  displayCancelIcon?: boolean;
+};
+
+const R = 25;
+
+const ProgressLoader = ({
+  progress = 0,
+  onPress,
+  displayCancelIcon = true,
+}: Props): React.ReactElement => {
+  const { colors } = useTheme();
+  const backgroundColor = colors.palette.primary.backgroundLight;
+  const progressColor = colors.palette.primary.dark;
+
+  const strokeWidth = 4;
```

Comment:
This doesn't seem to be matching the Figma, can you double-check the dimensions?
Also check with design in case we want to support multiple sizes 🤔 

![image](https://user-images.githubusercontent.com/4631227/130469821-5e3e6ca9-98e8-42ca-8fe4-dcc9f0769613.png)

Reply:
You are right, the stroke width is 2, not 4, thanks :pray: 
For the custom size, i assumed that this loader button would be used only in https://www.figma.com/file/wGzwuUVo0rkCJ3sM8cpbDY/Library---LLM?node-id=672%3A552 but it could be useful to support this indeed, i will check this.